src/js/pages/NetworkPage.js

```diff
@@ -8,37 +8,20 @@ import {Hooks} from 'PluginSDK';
 import Icon from '../components/Icon';
 import Loader from '../components/Loader';
 import Page from '../components/Page';
-import RouterUtil from '../utils/RouterUtil';
 import SidebarActions from '../events/SidebarActions';
-import TabsUtil from '../utils/TabsUtil';
 import TabsMixin from '../mixins/TabsMixin';
 
-let DEFAULT_NETWORK_TABS = {
-  '/networking/networks': {
-    content: 'Networks',
-    priority: 20
-  }
-};
-
-const DEFAULT_NETWORK_SUB_TABS = {};
+const NetworkingBreadcrumbs = () => {
+  const crumbs = [
+    <Link to="networking" key={-1}>Networks</Link>
```

Comment:
The negative key looks rather funky. Please use a different unique identifier, if possible. 

🗒  _Side Note: We should always prefer more specific keys over indexes to not only make the validator happy but actually improve the rendering performance._